_layouts/jurisdiction.html

```diff
@@ -11,6 +11,10 @@ <h1 class="post-title">{{ page.title }}</h1>
 
   <div class="row">
     <div class="col-md-6">
+    {% if page.jurisdiction = "ZA-FS" %}
+        {% include page.notice %}
```

Comment:
The page's metadata gets totally rewritten whenever we add new gazettes. Just cut and paste the notice directly into this file.